parcels/particleset.py

```diff
@@ -1130,13 +1130,13 @@ def _warn_particle_times_outside_fieldset_time_bounds(release_times: np.ndarray,
     if np.any(release_times):
         if np.any(release_times < time[0]):
             warnings.warn(
-                "Some particles are set to be released before the fieldset's first time and allow_time_extrapolation is set to False.",
+                "Some particles are set to be released before the fieldset's first time and the fields are not constant in time.",
```

Comment:
Is this still a correct warning? Should it not also check for the length of `time`?

Reply:
```suggestion
                "Some particles are set to be released outside the FieldSet's executable time domain.",
```